js/classes/entities/midground/weapon/slingshot.js

```diff
@@ -81,17 +87,19 @@ class Slingshot {
         this.playedStretchSound = false;
 
         // reset the shoot timer
-        this.shootTimer = Slingshot.SHOOT_DELAY;
+        this.timeSinceLastShot = 0;
+
+        // reset the reload timer
+        // this.timeSinceReload = 0;
```

Comment:
You have this comment written 3 lines above.

Reply:
Thanks for letting me know. I'll change it right away.

Reply:
Thanks for letting me know, but the statements are actually meant to be different from each other. this.timeSinceLastShot is used for delays between shots. this.timeSinceReload is commented out, but will soon be used for indicating the current level of charge for the slingshot.